indexer/indexer.go

```diff
@@ -194,6 +189,295 @@ func (ix *Indexer) handleRepoOp(ctx context.Context, evt *repomgr.RepoEvent, op
 	return nil
 }
 
+func (ix *Indexer) crawlAtUriRef(ctx context.Context, uri string) error {
```

Comment:
i just moved all this code to the bottom of the file. I want to delete it, but its used by the PDS still so we should probably just extract it to something pds specific (or just rewrite the pds logic better)